src/pages/home-page/HomePage.js

```diff
@@ -5,16 +5,21 @@ import "./home-page.scss";
 
 const HomePage = () => (
   <div className="home-page">
```

Comment:
If this div has the main content, you could replace the div with the `main` tag, the best practice is to avoid having a div wrapping everything, it's always better having semantic tag wrapping divs.